src/index.html

```diff
@@ -1,13 +1,324 @@
+                Influencer Marketing <br>
+                Product Placements <br>
+                Strategic Partnerships
+              </div>
+              <a href="#home" class="process__card-link">
+                <div class="process__card-btn"></div> Learn more
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="testimonials" id="testimonials">
+        <div class="testimonials__container">
+          <h5 class="testimonials__name">testimonials</h5>
+          <h3 class="testimonials__title"> What <br> people say </h3>
+          <div class="testimonials__cards">
+            <div class="testimonials__card">
+              <div class="testimonials__img">
+                <img src="images/testimonials/testimonial-ava-1.png" alt="Tal Gilad">
+              </div>
+              <div class="testimonials__quotes"></div>
+              <p class="testimonials__text">
+                AIR’s ideas are refreshing and out of the box. Authentic team that focuses
+                on the important path of the brand.
+              </p>
+              <h6 class="testimonials__card-name">Tal Gilad</h6>
+              <h5>Teach for America</h5>
+            </div>
+            <div class="testimonials__card">
+              <div class="testimonials__img">
+                <img src="images/testimonials/testimonial-ava-2.png" alt="Tal Gilad">
+              </div>
+              <div class="testimonials__quotes"></div>
+              <p class="testimonials__text">
+                AIR is an exceptional agency that leads with creative talent, first-class
+                account servicing.
+              </p>
+              <h6 class="testimonials__card-name">Azadeh Hawkins</h6>
+              <h5>Hawkins Consulting</h5>
+            </div>
+            <div class="testimonials__card">
+              <div class="testimonials__img">
+                <img src="images/testimonials/testimonial-ava-3.png" alt="Tal Gilad">
+              </div>
+              <div class="testimonials__quotes"></div>
+              <p class="testimonials__text">
+                AIR raises the agency bar to stratospheric heights on both creative output
+                and client service.
+              </p>
+              <h6 class="testimonials__card-name">Michel Grover</h6>
+              <h5>Hulu</h5>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="apply" id="apply">
+        <div class="apply__container">
+          <h2 class="apply__title">Vision, Passion, Results</h2>
+          <p class="apply__text"> We are sure that first-rate job is possible only if all three components are united.
+          </p>
+          <button class="apply__button">Learn more</button>
+        </div>
+      </section>
+    </main>
+    <footer class="footer" id="contacts">
+      <div class="footer__container">
+        <form
+          action="#home"
```

Comment:
```suggestion
          action="#"
```
You may simply just use `#`